bundles/org.eclipse.orion.client.debug/pom.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Copyright (C) 2013, SAP AG and others.
+
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    (http://www.eclipse.org/legal/epl-v10.html),
+    and the Eclipse Distribution License v1.0
+    (http://www.eclipse.org/org/documents/edl-v10.html).
+
+    Contributors:
+      Matthias Sohn - initial implementation
+-->
```

Comment:
Please use 2017 here. I'm not sure how the copyright should be written regarding IBM/SAP for the header and the contributors section and such. I'm assuming you just copy/pasted this from another `pom.xml` file and made some modifications?